.github/workflows/back-to-development.yaml

```diff
@@ -0,0 +1,54 @@
+name: Go back to Development
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+jobs:
+  post-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set RELEASED_VERSION_WITH_V variable from bufcli.go
+        # in the echo command, globbing patterns are used in the parameter substitution  not regex patterns
+        run: |
+          RELEASED_VERSION_WITH_V=`grep -oE "Version.*=.*\"[0-9]\.[0-9]+\.[0-9]+[^\"]*" private/buf/bufcli/bufcli.go`
+          echo "RELEASED_VERSION_WITH_V=v${RELEASED_VERSION_WITH_V##Version*=*\"}" >> $GITHUB_ENV
+      - name: Set NEXT VERSION variable from tag
+        run: |
+          NEXT_VERSION=$(echo ${{ env.RELEASED_VERSION_WITH_V }} | awk -F. -v OFS=. '{$NF += 1 ; print}')
+          echo "NEXT_VERSION=${NEXT_VERSION:1}-dev" >> $GITHUB_ENV
+      - name: Update version
+        run: make updateversion VERSION=${{ env.NEXT_VERSION }}
+      - name: Unrelease changelog
+        run: |
+          sed -i "/^# Changelog/ {
```

Comment:
This find the line that says `# Changelog` and inserts content two lines below this line. This isn't a complex expression, but newline and whitespace behavior seem to be different between gnu-sed and mac sed. Therefore, it is not added as a make target.

I wish we can update makego to make `SED_I` `gsed -i` on mac, that way we can guarantee consistent behavior across mac and linux. However, I suspect gsed isn't built-in to GitHub's mac runners and if a CI workflow running on mac needs to use gsed it would have to install it first.